OpenSim/Common/Reporter.h

```diff
@@ -177,7 +191,16 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
               const auto& value = chan.getValue(state);
               result[idx] = value;
         }
-        const_cast<Self*>(this)->_outputTable.appendRow(state.getTime(), result);
+        try {
+            const_cast<Self*>(this)->_outputTable.appendRow(state.getTime(),
+                                                            result);
+        } catch(const InvalidTimestamp&) {
+            OPENSIM_THROW(Exception,
+                          "Attempting to update reporter with rows having "
+                          "invalid timestamps. Hint: If running simulation in "
```

Comment:
The default message from `TimeSeriesTable` will be:
```
Timestamp at row <new-row-index> with value <new-timestamp> is less-than/equal to timestamp at row <previous-row> with value <previous-row-timestamp>.
```
I have changed throw statement to *append* to the message from the caught exception rather than replace it.